### PR TITLE
fix: --ignore and --only arguments

### PR DESCRIPTION
**Pull request by dangreen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
   "dependencies": {
     "commander": "^7.1.0",
     "fast-glob": "^3.2.5",
+    "micromatch": "^4.0.4",
```
Comment (dangreen) [inline package.json:46]: internally used by `fast-glob`

**Comment by dangreen** [comment]
@kdy1 Hi. I would appreciate if you reviewed it soon as possible.

**Comment by dangreen** [comment]
@kdy1 🙏

**Review by kdy1 — APPROVED** [review]
Thanks!

**Comment by dangreen** [comment]
@kdy1 Hi. How soon will you publish it?

**Comment by kdy1** [comment]
Oh, sorry. I forgot it.
I published a new version.

**Pull request merged**